packages/form-builder/addon/components/cfb-code-editor.js

```diff
@@ -2,11 +2,14 @@ import { action } from "@ember/object";
 import { addObserver } from "@ember/object/observers";
 import Component from "@glimmer/component";
 import { CodeJar } from "codejar";
-import Prism from "prismjs";
+import hljs from "highlight.js/lib/core";
+import json from "highlight.js/lib/languages/json";
+import markdown from "highlight.js/lib/languages/markdown";
+import jexl from "highlightjs-jexl/src/languages/jexl";
```

Comment:
Can we try to get this into `hightlighjs`?

Reply:
Highlight doesn’t accept new languages in its core anymore. One can make PRs that list a new language as supported via a plugin though - I didn’t do that yet because our plugin is still lacking tests. Would love to do that though, but I might not find the time.

Reply:
Alright, I didn't know that.